src/models/whatsnewmodel.cpp

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "whatsnewmodel.h"
+#include "leakdetector.h"
+#include "logger.h"
+#include "mozillavpn.h"
+#include "settingsholder.h"
+#include "featurelist.h"
+#include "feature.h"
+
+#include <QList>
+
+namespace {
+Logger logger(LOG_MODEL, "WhatsNewModel");
+}
+
+WhatsNewModel::WhatsNewModel() {
+  MVPN_COUNT_CTOR(WhatsNewModel);
+
+  WhatsNewModel::setNewFeatures();
+}
+
+WhatsNewModel::~WhatsNewModel() { MVPN_COUNT_DTOR(WhatsNewModel); }
+
+int WhatsNewModel::featureCount() { return m_featurelist.size(); }
+
+QHash<int, QByteArray> WhatsNewModel::roleNames() const {
+  QHash<int, QByteArray> roles;
+  roles[RoleId] = "id";
```

Comment:
This has a lot of duplicate data. Can you check if you do:
`roles[RoleFeature] = "feature";`

Then, expose the Feature object directly.